editor/package.json

```diff
@@ -10,40 +10,40 @@
     "format": "prettier --write ."
   },
   "dependencies": {
-    "@auth0/auth0-react": "^2.2.4",
-    "@turf/center": "^7.1.0",
-    "@types/lodash": "^4.17.13",
-    "@types/mapbox-gl": "^3.4.0",
+    "@auth0/auth0-react": "^2.3.0",
+    "@turf/center": "^7.2.0",
+    "@types/lodash": "^4.17.15",
+    "@types/mapbox-gl": "^3.4.1",
     "@types/papaparse": "^5.3.15",
     "bootstrap": "^5.3.3",
     "date-fns": "^4.1.0",
-    "graphql": "^16.9.0",
-    "graphql-request": "^7.1.0",
+    "graphql": "^16.10.0",
+    "graphql-request": "^7.1.2",
     "immer": "^10.1.1",
     "lodash": "^4.17.21",
-    "mapbox-gl": "^3.7.0",
-    "next": "14.2.13",
-    "papaparse": "^5.4.1",
-    "react": "^18",
-    "react-bootstrap": "^2.10.4",
-    "react-datepicker": "^7.5.0",
-    "react-dom": "^18",
-    "react-hook-form": "^7.53.2",
+    "mapbox-gl": "^3.9.4",
+    "next": "14.2.x",
```

Comment:
i am enabling floating patch releases from nextjs. we could also opt for floating minor releases, but i'm not sure if we want to take that step just yet. I would definitely like to bake in an `npm audit` and `npm update` into our release process.